### ISSUE TITLE
Key "Principled BSDF" not found

### ISSUE THREAD

**Issue by zetanumbers** [lennart-g/blender-md2-importer#6]
Encountered an error during md2 model import

<details>

<summary> Error message </summary>

<img width="891" alt="Снимок экрана 2023-11-13 в 18 17 55" src="https://github.com/lennart-g/blender-md2-importer/assets/53300196/b00a3d5b-e008-465c-8fcb-455868f35239">

</details>

Source of error:

https://github.com/lennart-g/blender-md2-importer/blob/4705481479f1e21fe278a8f6a66c084f71ebace6/io_import_md2/blender_load_md2.py#L129

<details>

<summary> blender --version </summary>

```
Blender 3.6.5
build date: 2023-10-18
build time: 23:49:48
build commit date: 2023-10-16
build commit time: 14:30
build hash: cf1e1ed46b7e
build platform: Darwin
build type: release
build c flags:  -Wall -Werror=implicit-function-declaration -Werror=return-type -Wno-tautological-compare -Wno-unknown-pragmas -Wno-char-subscripts -Wstrict-prototypes -Wmissing-prototypes -Wunused-parameter -Werror=unguarded-availability-new  -mmacosx-version-min=11.00 -Xclang -fopenmp -I'/Users/blender/git/blender-v360/blender.git/../lib/darwin_arm64/openmp/include' -std=gnu11  -pipe -funsigned-char -fno-strict-aliasing -ffp-contract=off  
build c++ flags:  -Wall -Wc++20-designator -Wno-tautological-compare -Wno-unknown-pragmas -Wno-char-subscripts -Wno-overloaded-virtual -Wno-sign-compare -Wno-invalid-offsetof -Wno-suggest-override -Werror=unguarded-availability-new  -mmacosx-version-min=11.00 -ftemplate-depth=1024 -stdlib=libc++ -Xclang -fopenmp -I'/Users/blender/git/blender-v360/blender.git/../lib/darwin_arm64/openmp/include'  -pipe -funsigned-char -fno-strict-aliasing -ffp-contract=off  
build link flags: -fexceptions -framework CoreServices -framework Foundation -framework IOKit -framework AppKit -framework Cocoa -framework Carbon -framework AudioUnit -framework AudioToolbox -framework CoreAudio -framework Metal -framework QuartzCore -framework Accelerate -framework ForceFeedback -framework GameController -framework CoreHaptics -liconv -Wl,-unexported_symbols_list,'/Users/blender/git/blender-v360/blender.git/source/creator/symbols_apple.map' -stdlib=libc++
build system: CMake
```

</details>

Same as https://github.com/armory3d/armory/issues/1407
Fixed with https://github.com/armory3d/armory/commit/ced8cd845fa98893cb870ae628b7e328b25dcb2e

**Comment by lennart-g** [comment]
Hi @ZetaNumbers,

I'm not familiar with that library. Is the issue you encountered a bug in this md2 importer plugin or in the blender Armory add-on? 

The fix you linked appears to be included in [Armory Release 21.06](https://github.com/armory3d/armory/releases/tag/21.06) so I assume the problem does not appear with recent Armory versions anymore?

**Comment by zetanumbers** [comment]
No, I don't use armory, it only had a similair issue. Sorry for the confusion. I've added additional info on encountered error. However, i can no longer reproduce it, so i close it for now.
